### DISCUSSION TITLE
documentation  on how to implement custom connectors

### DISCUSSION THREAD

**Discussion by peterclemenko** [redpanda-data/connect#2849] [General]
it would be nice if there were documentation on how to implement custom connectors

**Comment by mihaitodor** [comment]
Hey @peterclemenko you can see some examples here https://github.com/redpanda-data/redpanda-connect-plugin-example and here https://pkg.go.dev/github.com/redpanda-data/benthos/v4/public/service

PS: Moving to a discussion as per #2026
